omero/sysadmins/public.txt

```diff
@@ -0,0 +1,135 @@
+automatically logged in and will be able to access the data available to the
+defined 'public user'.
+
+To set this up on your OMERO.web installation:
+
+- Create a group with read-only permissions (the name can be anything e.g.
+  "public-data"). We recommend read-only permissions so that the public user
+  will not be able to modify, delete or annotate data belonging to other
+  members.
+
+- Create a member of this group, noting the username and password (you will
+  enter these below). Again, the First Name, Last Name, username and
+  password can be anything you like.
+
+  .. note:: If you add this member to other groups, all data in these groups
+      will also become publicly accessible for as long as this user remains
+      in the group.
+
+- Enable the :property:`omero.web.public.enabled` property and set
+  :property:`omero.web.public.user` and
+  :property:`omero.web.public.password`:
+
+  ::
+
+     $ bin/omero config set omero.web.public.enabled True
+
+     $ bin/omero config set omero.web.public.user '<username>'
+
+     $ bin/omero config set omero.web.public.password '<password>'
+
+- Set the :property:`omero.web.public.url_filter`. This filter is a
+  regular expression that will allow only matching URLs to be accessed
+  by the public user.
+
+  There are three common use cases for the URL filter:
+
+  - Enable 'webgateway' URLs which includes everything needed for the
+    full image viewer:
+
+    ::
+
+       $ bin/omero config set omero.web.public.url_filter '^/webgateway'
+
+
+    without the ability to download data:
+
+    ::
+
+       $ bin/omero config set omero.web.public.url_filter '^/webgateway/(?!archived_files|download_as)'
+
+
+    Then you can access public images via the following link
+    `\http://your_host/webgateway/img_detail/IMAGE_ID/`.
+
+  - Create your own public pages in a separate app
+    (see :doc:`create app </developers/Web/CreateApp>`) and allow
+    public access to that app. For example, to allow only
+    URLs that start with '/my_web_public' you would use:
+
+    ::
+
+       $ bin/omero config set omero.web.public.url_filter '/my_web_public'
+
+
+  - You can use the full webclient UI for public browsing of images.
+    However, the webclient UI was not designed for public use and allows
+    various actions that create data or are resource intensive.
+    These can be selectively disabled using the following command:
+
+    ::
+
+       $ bin/omero config set omero.web.public.url_filter '^/(?!webadmin|webclient/(action|logout|annotate_(file|tags|comment|rating|map)|script_ui|ome_tiff|figure_script)|webgateway/(archived_files|download_as))'
+
+- Set the :property:`omero.web.public.server_id` which the public user will be
+  automatically connected to. Default: 1 (the first server in the
+  :property:`omero.web.server_list`)
+
+  ::
+
+     $ bin/omero config set omero.web.public.server_id 1
+
+If you wish to mix public and restricted access to the system, a registered
```

Comment:
This might need a bit more explanation. E.g.
"If you enable public access to the main "webclient" pages but still wish to log in as a registered user, you will need to go directly to the login page at `\https://your_host/webclient/login/`.